cmake/legateboost/get_rapids_cmake.cmake

```diff
@@ -17,7 +17,7 @@ macro(legateboost_get_rapids_cmake)
 
   if(NOT rapids-cmake-version)
     # default
-    set(rapids-cmake-version 24.08)
+    set(rapids-cmake-version 24.10)
     set(rapids-cmake-sha "3cc764f287a6f3caeee5dd1c96c24b1710d4cdf1")
```

Comment:
I don't know from memory... does the version matter if we're still pinning to a specific commit of `rapids-cmake`?

I can't tell what version is getting downloaded from the build logs since they're pretty quiet: https://github.com/rapidsai/legate-boost/actions/runs/12374421493/job/34536930592?pr=195

I'd support updating their verbosity. Not sure the right place...maybe `PIP INSTALL_ARGS` should pick up `-v` in `build.sh`? Or maybe we need to set CMake verbosity in `scikit-build-core`: https://scikit-build-core.readthedocs.io/en/latest/configuration.html#verbosity. Or maybe both?

Reply:
Ha ok thanks! I think it's great that we're getting this CMake configure output now in CI logs... a lot of solutions to bugs are found in those logs in my experience:

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/49fbe435-2780-4003-bfbe-eb7d4efe91b3" />

If just removing the sha works then I think we're good to merge this 😁